src/index.ts

```diff
@@ -197,6 +198,20 @@ async function appBoot() {
 		console.log( `System locale: ${ app.getSystemLocale() }` );
 		console.log( `Used language: ${ locale }` );
 
+		// Start the proxy server for custom domains
+		try {
+			const proxyStarted = await startProxyServer();
+			if ( proxyStarted ) {
+				console.log( 'Custom domain proxy server started successfully' );
+			} else {
+				console.warn(
+					'Failed to start custom domain proxy server - custom domains will require port numbers'
+				);
+			}
+		} catch ( error ) {
+			console.error( 'Error starting proxy server:', error );
+		}
```

Comment:
To accommodate the `EADDRINUSE` error modal, I recommend moving this logic below the `createMainWindow` call. Otherwise, the error modal will display before we create the main window, which looks odd.